check_if_image_tag_exists/main.py

```diff
@@ -33,18 +33,26 @@ def check_if_tag_exists(raw_image_path):
         print tag
 
     if image_tag in existing_tags:
-        sys.exit("Tag already exists in remote repository! Exiting build.")
-    print "Tag does not exist in remote repository! Continuing with build."
+        print "Tag \'" + image_tag + "\' already exists in remote repository!"
```

Comment:
sure, the escaping is for the single quotes though (so they appear in the string).

I thought `Tag 'latest' already exists...` was a bit more clear than `Tag latest already exists...`